### PR TITLE
Add cards for SM EWK production of W+h plus two jets 

### PR DESCRIPTION
**Pull request by DickyChant**
Dear Gen Experts,

This PR is created for our analysis on studying the Wh production with VBF topology. You can find a detailed talk vai [https://indico.cern.ch/event/1109096/#23-studies-for-vh-bb2-jet-proc](https://indico.cern.ch/event/1109096/#23-studies-for-vh-bb2-jet-proc). 

The run card is modified from https://github.com/cms-sw/genproductions/tree/7368216b51aad57a26043fb33daec61e5592abe2/bin/MadGraph5_aMCatNLO/cards/WW_leptonic, we basically keep all cuts except for loosing mjj to 0, we also change the maxjetflavor to 4 to meet the requirement of 4f scheme.

### CODE REVIEW

**Comment by DickyChant** [comment]
In our analysis, the higgs will decay to b+bbar via Pythia. This is one of the standard treatment of higgs decay. Corresponding fragments can be found in either [genproduction area](https://github.com/cms-sw/genproductions/blob/master/genfragments/ThirteenTeV/Higgs/ttHJetTobb_M125_13TeV_amcatnloFXFX_madspin_pythia8_cff.py) or [official samples](https://cms-pdmv.cern.ch/mcm/requests?prepid=HIG-RunIIFall17wmLHEGS-04148&shown=262271).

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WtoLNuhjjEWK/WtoLNuhjj_EWK_4f_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 0        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 5        = asrwgtflavor     ! highest quark flavor for a_s reweight
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WtoLNuhjjEWK/WtoLNuhjj_EWK_4f_run_card.dat:72]: please change to 4= asrwgtflavor to keep the flavor consistent

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WtoLNuhjjEWK/WtoLNuhjj_EWK_4f_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+ -1  = drjlmax ! max distance between jet and lepton
+ -1  = drabmax ! max distance between gamma and b
+ -1  = drblmax ! max distance between b and lepton
+ -1  = dralmax ! maxdistance between gamma and lepton
+#*********************************************************************
+# Minimum and maximum invariant mass for pairs                       *
+# WARNING: for four lepton final state mmll cut require to have      *
+#          different lepton masses for each flavor!                  *           
+#*********************************************************************
+ 0 = mmjj    ! min invariant mass of a jet pair
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WtoLNuhjjEWK/WtoLNuhjj_EWK_4f_run_card.dat:171]: i prefer 100=mmjj here as other vbs process used

Comment (DickyChant) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/WtoLNuhjjEWK/WtoLNuhjj_EWK_4f_run_card.dat]: We have no objection to this, modified in the incoming commit.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WtoLNuhjjEWK/WtoLNuhjj_EWK_4f_run_card.dat (hunk 1 above)
Comment (DickyChant) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/WtoLNuhjjEWK/WtoLNuhjj_EWK_4f_run_card.dat]: Thanks! Changed in the incoming commit.

**Pull request merged**